routers/repo/hashtag.go

```diff
@@ -0,0 +1,49 @@
+package repo
+
+import (
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/base"
+	"strings"
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/log"
+	"net/http"
+)
+
+const (
+	HASHTAGS    base.TplName = "repo/hashtag/list"
+)
+
+// Produce a page listing all hashtags for this language, with counts
```

Comment:
As per Go convention, docs for a function should start with the function name (see [here](https://blog.golang.org/godoc-documenting-go-code))